src/components/app/FooterLinks.js

```diff
@@ -38,23 +39,23 @@ export class FooterLinks extends PureComponent<{||}, State> {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Legal
+          <Localized id="FooterLinks--legal">Legal</Localized>
```

Comment:
General note: I think the general approach is not leave any text in the markup, and let it be managed by Fluent completely. With text, you would risk flashing untranslated text unnecessarily.

Reply:
@canova I guess we discussed keeping the texts as it was in the markup for now, what do you suggest on that?

Reply:
I think @flodolo has more experience working on this, so I'd be inclined to follow his suggestion :-) even if we thought otherwise before.

Reply:
We discussed this on our internal channel, and now we think we'll keep it this way for now. The main reason is that our tests don't have the FTL file properly setup yet, and all our tests will likely fail with the blank fallback strings.

We'll probably revisit this in the future, but this is the easiest path forward to move this PR to the finish line. I hope this compromise is good enough for you @flodolo :)

Reply:
Yep, as long as it's tracked for future improvement, I'm fine.

Reply:
yep, we'll file a bug. Thanks!

Reply:
Filed #3183